From 4367e1333f43e0e07bee151412b1875f72d144ed Mon Sep 17 00:00:00 2001
From: Subu Dwevedi <messigoatcr7nop@gmail.com>
Date: Thu, 18 Dec 2025 16:03:18 +0530
Subject: [PATCH 01/15] asahi

Signed-off-by: Subu Dwevedi <messigoatcr7nop@gmail.com>
---
 drivers/mfd/macsmc.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/drivers/mfd/macsmc.c b/drivers/mfd/macsmc.c
index e3893e255ce5..85410f357603 100644
--- a/drivers/mfd/macsmc.c
+++ b/drivers/mfd/macsmc.c
@@ -45,8 +45,11 @@
 #define SMC_TIMEOUT_MS		500
 
 static const struct mfd_cell apple_smc_devs[] = {
+	MFD_CELL_NAME("macsmc-input"),
 	MFD_CELL_OF("macsmc-gpio", NULL, NULL, 0, 0, "apple,smc-gpio"),
+	MFD_CELL_OF("macsmc-hwmon", NULL, NULL, 0, 0, "apple,smc-hwmon"),
 	MFD_CELL_OF("macsmc-reboot", NULL, NULL, 0, 0, "apple,smc-reboot"),
+	MFD_CELL_OF("macsmc-rtc", NULL, NULL, 0, 0, "apple,smc-rtc"),
 };
 
 static int apple_smc_cmd_locked(struct apple_smc *smc, u64 cmd, u64 arg,
-- 
2.43.0

