From 0add1d471e822455d69e02bc7e79f4392f977a96 Mon Sep 17 00:00:00 2001
From: Subu Dwevedi <messigoatcr7nop@gmail.com>
Date: Mon, 22 Dec 2025 21:20:16 +0530
Subject: [PATCH 13/15] mfd/macsmc_core: add new drivers to apple_smc_devs

Signed-off-by: Subu Dwevedi <messigoatcr7nop@gmail.com>
---
 drivers/mfd/macsmc-core.c | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/drivers/mfd/macsmc-core.c b/drivers/mfd/macsmc-core.c
index f0920bef7a8b..b5cc3082a344 100644
--- a/drivers/mfd/macsmc-core.c
+++ b/drivers/mfd/macsmc-core.c
@@ -12,9 +12,14 @@
 #include <linux/spinlock.h>
 
 static const struct mfd_cell apple_smc_devs[] = {
+	MFD_CELL_NAME("macsmc-accel"),
+	MFD_CELL_NAME("macsmc-light"),
+	MFD_CELL_NAME("macsmc-battery"),
+	MFD_CELL_NAME("macsmc-bl"),
+	MFD_CELL_NAME("macsmc-chamshell"),
 	MFD_CELL_NAME("macsmc-input"),
 	MFD_CELL_OF("macsmc-gpio", NULL, NULL, 0, 0, "apple,smc-gpio"),
-	MFD_CELL_OF("macsmc-hwmon", NULL, NULL, 0, 0, "apple,smc-hwmon"),
+	MFD_CELL_NAME("macsmc-hwmon"),
 	MFD_CELL_OF("macsmc-reboot", NULL, NULL, 0, 0, "apple,smc-reboot"),
 	MFD_CELL_OF("macsmc-rtc", NULL, NULL, 0, 0, "apple,smc-rtc"),
 };
-- 
2.43.0

