From 06d49a8daafe72893fd68180b0c4aa8473523b91 Mon Sep 17 00:00:00 2001
From: Subu Dwevedi <messigoatcr7nop@gmail.com>
Date: Mon, 22 Dec 2025 22:00:27 +0530
Subject: [PATCH 14/15] acpi/sbs: add quirk to fix duplicate battery issue

Signed-off-by: Subu Dwevedi <messigoatcr7nop@gmail.com>
---
 drivers/acpi/sbs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/acpi/sbs.c b/drivers/acpi/sbs.c
index f91f45a90704..0635ea4cfd8b 100644
--- a/drivers/acpi/sbs.c
+++ b/drivers/acpi/sbs.c
@@ -641,6 +641,7 @@ static int acpi_sbs_add(struct acpi_device *device)
 	if (!strncmp(product_name, "MacBook", 7))
 		return -ENODEV;
 #endif
+
 	sbs = kzalloc(sizeof(struct acpi_sbs), GFP_KERNEL);
 	if (!sbs) {
 		result = -ENOMEM;
-- 
2.43.0

